.coveragerc

```diff
@@ -0,0 +1,8 @@
+[report]
+include =
+    *march_rqt_gait_selection*
+    *march_rqt_input_device*
+    *march_rqt_note_taker*
+omit =
+    *test*
```

Comment:
multiple example show that *test* is not included in the coverage report. Isn't it useful to check whether the coverage of your test is 100%?

Reply:
The coverage of the tests are now also included but are not used in the PR comment, commit status and badge :tada: